### PR TITLE
Modify getting-started document to reflect our new Azure project structures, testing, etc...

### PR DESCRIPTION
**Pull request by BradySkylight**
There are places where we still need to do work and they are marked by a ??TODO  or TODO??.

### CODE REVIEW

**Review by DanPaseltiner — APPROVED** [review]
This is great. A couple minor things I think we should address before merging, but no need for them to be blocking. Once we get a little more familiar with somethings in Azure like ADF we'll need to do another revision on this doc.

File: docs/README.md
Comment (BradySkylight) [reply docs/README.md:53]: Yup, no problem.  Will do that now.

File: docs/getting_started.md
Comment (BradySkylight) [reply docs/getting_started.md:3]: Love this!  Great idea!

File: docs/getting_started.md
Comment (BradySkylight) [reply docs/getting_started.md:102]: I pulled this from the previous azure project, so we may not need it.  I'll remove it for now, but we should review this doc again after we are done creating all the BBs.

File: docs/README.md
Comment (BradySkylight) [reply docs/README.md:73]: https://app.clickup.com/t/327fry3

File: docs/getting_started.md
Comment (BradySkylight) [reply docs/getting_started.md:147]: Interesting.  I was just going off of what we actually have within the project itself.

**Pull request merged**
